src/products/android-l10n/review_Android_strings.md

```diff
@@ -0,0 +1,41 @@
+# Reviewing strings for a new release of Mozilla Android products
+
+Usually, a new release of our Android products means an update to strings. If this is the case, pull requests will be made throughout the release cycle by the mobile team, in order to land these new strings in the [android-l10n](https://github.com/mozilla-l10n/android-l10n) repository in `mozilla-l10n`.
+
+The PRs (also marked as `quarantine`) are reviewed by an l10n-driver - most often by the PM in charge of the project.
+
+Let’s go over some of the steps needed over time in order to review strings correctly for a new release.
+
+## Reviewing the PR
+
+Let's consider Firefox for Android as an example - in fact, these apply to other Android products as well, and they also follow the same train schedule. During the [Android l10n cycle](https://docs.google.com/spreadsheets/d/1hER_w7pOsWSjeBaUqMd_9ClYCCGlwc4jaFYpcBvFE8g/edit#gid=1960046531), the Android team will make pull requests, in order to land new strings for the upcoming release.
+
+Let’s consider a past PR [here](https://github.com/mozilla-l10n/android-l10n/pull/391/files).
+
+The first file present is `_meta/mozilla-mobile-fenix.json`. Updates to the `main` section are expected and can be ignored.
```

Comment:
Why are they expected? What does that file track? ;-)

Reply:
It looks like some kind of automation for android-l10n: https://github.com/mozilla-l10n-automation-bot/android-l10n/blame/master/_meta/mozilla-mobile-fenix.json
I honestly don't know more, though

Reply:
Part of writing documentation is learning more around things that are initially not clear ;-) 

That is indeed the hash of a [changeset](https://github.com/mozilla-mobile/fenix/commit/76c0c4ad1425cfd79c25d707921e8185620ad080) from the fenix repository. It's the changeset that is used to generate the strings in that export/PR

Reply:
Makes total sense :)
Thanks for detailing. I'll update this as well